app/code/Magento/Customer/Model/Customer.php

```diff
@@ -486,7 +492,8 @@ public function getAddressById($addressId)
      * Getting customer address object from collection by identifier
      *
      * @param int $addressId
-     * @return Address
+     * @return \Magento\Framework\DataObject
```

Comment:
Isn’t it returns Address object? Looks like incorrect change

Reply:
Yes, it returns Address object.